cmd/staking.go

```diff
@@ -53,7 +53,7 @@ $ lens tx staking delegate cosmosvaloper1sjllsnramtg3ewxqwwrwjxfgc4n4ef9u2lcnj0
 
 			msg := types.NewMsgDelegate(delAddr, sdk.ValAddress(valAddr), amount)
 
-			res, ok, err := cl.SendMsg(cmd.Context(), msg)
+			res, ok, err := cl.SendMsg(cmd.Context(), msg, delAddr.String())
```

Comment:
You can't use the `string` functions need to use the `client.RenderAddress` stuff to avoid bech32 weirdness

Reply:
Also send msg shouldn't take that third arg...